src/components/deltabargraph.js

```diff
@@ -1,14 +1,19 @@
 import * as d3 from 'd3';
 import React, {useEffect, useRef, useState} from 'react';
 
-function DeltaBarGraph({timeseries, arrayKey}) {
+function DeltaBarGraph({timeseries, arrayKeyProp}) {
   const [data, setData] = useState([]);
+  const [arrayKey, setArrayKey] = useState(`daily${arrayKeyProp}`);
```

Comment:
I think it might make sense to directly use the prop, since storing it inside a state is not really required.